src/Components/__stories__/ArtworkMetadata.story.tsx

```diff
@@ -1,6 +1,6 @@
 import { storiesOf } from "@storybook/react"
 import * as React from "react"
-import * as Relay from "react-relay"
+import * as Relay from "react-relay/classic"
```

Comment:
:OG:

Reply:
Still getting some ❌ , but I think thats from the WIP branch stuff.

Either way, I think we discussed a larger refactor with this component and the overlay/extended stuff, so feel free to merge, or wait, etc.

Reply:
Yeah the branch itself is not green. If it does what you intended then I’ll merge now 👍